File: backend/spring/src/main/java/at/gradwohl/website/model/lieferbar/Lieferbar.java
package at.gradwohl.website.model.lieferbar;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@Table(name = "Lieferbar")
public class Lieferbar {
    @EmbeddedId
    private LieferbarId id;

    @Column(name = "LF_Montag")
    private boolean montag;

    @Column(name = "LF_Dienstag")
    private boolean dienstag;

    @Column(name = "LF_Mittwoch")
    private boolean mittwoch;

    @Column(name = "LF_Donnerstag")
    private boolean donnerstag;

    @Column(name = "LF_Freitag")
    private boolean freitag;

    @Column(name = "LF_Samstag")
    private boolean samstag;

    @Column(name = "LF_Sonntag")
    private boolean sonntag;
}

File: frontend/gradwohl_website/src/model/lieferbar/lieferbar.ts
import { lieferbarId } from "./lieferbarId"

export interface lieferbar{
    id: lieferbarId,
    montag: boolean,
    dienstag: boolean,
    mittwoch: boolean,
    donnerstag: boolean,
    freitag: boolean,
    samstag: boolean,
    sonntag: boolean
}